package.json

```
@@ -9,8 +9,8 @@
   "types": "dist/index.d.ts",
   "scripts": {
     "postinstall": "run-s build",
-    "compile": "git clone --branch mvp0.1 https://github.com/alastria/alastria-identity.git && node src/configFile.js && tsc",
-    "clean": "rimraf dist",
+    "compile": "git clone --branch develop https://github.com/alastria/temp-alastriaID-truffle-contracts.git; npm install --prefix temp-alastriaID-truffle-contracts; npm run compile --prefix temp-alastriaID-truffle-contracts; node src/configFile.js; tsc",
```

```suggestion
    "compile": "git clone --branch develop https://github.com/alastria/temp-alastriaID-truffle-contracts.git && npm install --prefix temp-alastriaID-truffle-contracts && npm run compile --prefix temp-alastriaID-truffle-contracts && node src/configFile.js && tsc",
```

I think we must use `&&` instead of `;`. If we use `;` seems that windows users have some problems. https://github.com/alastria/alastria-wallet/issues/86#issuecomment-735755647

---

Done!